Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/StructurePanel.java

```diff
@@ -582,7 +583,7 @@ private Collection<View> buildStructureTreeRecursively(LogicalDivision structure
          * appends it to the parent as a child. That’s the logic of the JSF
          * framework. So you do not have to add the result anywhere.
          */
-        DefaultTreeNode parent = new DefaultTreeNode(node, result);
+        DefaultTreeNode parent = new DefaultTreeNode("Structure", node, result);
```

Comment:
Could the String `Structure` be saved to a constant that is referenced here (and elsewhere, if required) instead?